assets/js/editor.js

```diff
@@ -321,6 +322,7 @@ $gp.editor = (
 					success: function( data ) {
 						$gp.notices.success( 'Saved!' );
 						$gp.editor.replace_current( data );
+						$gp.editor.next();
```

Comment:
Jumping to the next editor is a bad idea because 1) a translation can have more than one warning (see #370) and 2) a translation can be in a pending state.

In both steps you'd have to re-open the previous editor for further status changes, that sounds like a step backward to me.